index.js

```diff
@@ -146,7 +133,40 @@ function resolveWebpackShim(source, kibanaPath, rootPath) {
   return kibanaFileMatches;
 }
 
-exports.interfaceVersion = 2
+/*
+ * Resolves global import aliases that map a prefix to a Kibana source directory
+ * Used for UI: https://github.com/elastic/kibana/blob/5c04ff65fbb3b16f8958f8241488463136415670/src/ui/ui_bundler_env.js#L29
+ * Used for tests: https://github.com/elastic/kibana/blob/5c04ff65fbb3b16f8958f8241488463136415670/src/core_plugins/tests_bundle/index.js#L70-L75
+ */
+function resolveAliasModuleImport(source, kibanaPath) {
+  debug(`resolving ui import: src/core_plugins/dev_mode/public/${source}`);
+  const checkPaths = [
```

Comment:
Yeah, I was thinking about looking at how a different resolver module does tests. I hope we don't have to mock `path` or something